common4j/src/testFixtures/java/com/microsoft/identity/common/nativeauth/ApiConstants.kt

```diff
@@ -30,8 +30,8 @@ import java.net.URL
  */
 interface ApiConstants {
     companion object {
-        const val BASEPATH = "https://native-auth-mock-api.azurewebsites.net/"
-        private const val BASE_REQUEST_PATH = BASEPATH + "1234/"
+        val BASEPATH: String? by lazy { System.getenv("MOCK_API_URL") }
```

Comment:
This is not how we should be dealing with env vars. They should be set as part of the gradle script, in BuildConfig vars. E.g. look at how the Labs API secret is set in various build.gradle files: `buildConfigField("String", "LAB_CLIENT_SECRET", "\"$labClientSecret\"")`.

Reply:
Let me know if you need some help with this.